test/parallel/test-http-socket-listeners.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
Please use a more descriptive test name and/or clarify what the test does in a comment.

Reply:
👍